.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: csharp
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
Haha, if you look at this list I'm not sure they can fail more frequently: https://travis-ci.org/aspnet/KestrelHttpServer/pull_requests